### PR TITLE
Fix testcase IP_UDP_I&M_EPM

### PR DESCRIPTION
**Pull request by olbjo**
Fixes IP_UDP_I&M_EPM together with #219 and #223

### CODE REVIEW

File: src/device/pf_cmina.c [hunk 1]
```diff
@@ -1341,13 +1342,13 @@ void pf_cmina_show (pnet_t * net)
       net->cmina_current_dcp_ase.station_name);
    printf ("\n");
 
-   printf ("Default device_vendor          : <%s>\n", p_cfg->device_vendor);
+   printf ("Default device_vendor          : <%s>\n", p_cfg->product_name);
```
Comment (pyhys) [inline src/device/pf_cmina.c:1345]: "Default product_name ..." also in two following printf lines.

File: src/pf_types.h [hunk 2]
```diff
@@ -781,7 +781,13 @@ typedef struct pf_eth_frame_id_map
 typedef struct pf_cmina_dcp_ase
 {
    char station_name[PNET_STATION_NAME_MAX_LEN + 1]; /* Terminated */
-   char device_vendor[20 + 1];                       /* Terminated */
+
+   /*
+    * DPC DeviceVendorValue is set to configured product_name
```
Comment (pyhys) [inline src/pf_types.h:786]: DCP

**Review by pyhys — APPROVED** [review]
Great! Just a few documentation changes.

**Review by pyhys — APPROVED** [review]
Please remove `device_vendor` from `include/pnet_api.h`, `sample_app/sampleapp_common.c` and `test/utils_for_testing.cpp`

File: src/device/pf_cmina.c (hunk 1 above)
Comment (olbjo) [reply src/device/pf_cmina.c:1345]: done

File: src/pf_types.h (hunk 2 above)
Comment (olbjo) [reply src/pf_types.h:786]: done

**Comment by olbjo** [comment]
Also removed device_vendor

**Pull request merged**